clippy_lints/src/methods/option_map_unwrap_or.rs

```diff
@@ -27,6 +29,7 @@ pub(super) fn check<'tcx>(
     unwrap_recv: &rustc_hir::Expr<'_>,
     unwrap_arg: &'tcx rustc_hir::Expr<'_>,
     map_span: Span,
+    msrv: &Msrv,
```

Comment:
NIT: Can you add this lint to the configuration documentation, to make sure that it's correctly linked in Clippy's lint list?

You should be able to just add it to the list here:

https://github.com/rust-lang/rust-clippy/blob/ea4c5c5ca55ae9614552c0d285ee6c051cf88b30/clippy_lints/src/utils/conf.rs#L297-L300

(On your branch, that is just a link to the file :))